### PR TITLE
CSS fixes

### LINKED ISSUE #192: Print style does not work anymore
I guess I broke it by I am not sure..

### LINKED ISSUE #193: Flex box change broke full-screen

### PR DESCRIPTION
**Pull request by willdurand**
- [x] fix #192
- [x] fix #193 

These changes are required since https://github.com/TailorDev/monod/commit/26e7c58173b4286b61f8ff72759df1e9f2b76c32.

### CODE REVIEW

**Comment by bebatut** [comment]
It does print more than the first page on long document :(

**Comment by bebatut** [comment]
I works !!! 🎉

File: app/scss/_print.scss [hunk 1]
```diff
@@ -65,6 +72,7 @@
     pre, blockquote {
         border: 0;
         border-left: 3px solid $silver;
+        margin-bottom: 11pt !important;
```
Comment (jmaupetit) [inline app/scss/_print.scss]: Why do you use `pt` unit here?

Comment (willdurand) [inline app/scss/_print.scss]: Because it is the unit used for paragraphs, and I wanted a one-(paragraph-)line margin.

**Pull request merged**